assets/constants.js

```diff
@@ -16,6 +16,9 @@ var constants = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
 
+  // This specifies the current editor in use and used to switch between CK4 & CK5.
+  "IS_CURRENT_RTE_CKEDITOR_4" : true,
```

Comment:
CURRENT_RTE_IS_CKEDITOR_4

(we usually start names with verbs only when they are methods/functions)

Reply:
Done Thanks!